### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by TheSoundCoder** [TheSoundCoder/GameChatBalancer#5] [General]
Please feel free to provide feedback to the project.
- What do you like?
- What should be improved?
- Did you build your own GameChatBalancer?
  - Does it work?
  - Do you need help?

**Comment by DrewBrummett** [comment]
I can't wait to start trying to build this! I was so sad when my chatmix broke out of nowhere cause it was one of the most useful things ever.

I was curious how hard it would be to implement a bluetooth module for wireless audio?!

Keep up the work! Hopefully by building it I can be of some use.

**Reply by TheSoundCoder** [reply]
Hi Drew!
I am very happy to see that other people like the project. If you build one, please provide me feedback if anything is not described well or does not work.

Regarding bluetooth / wireless audio: GameChatBalancer should work with all kind of audio devices which are connected to your Windows PC. As GameChatBalancer uses the Windows mixer it should work out of the box.

Hope this answer helps!

**Reply by DrewBrummett** [reply]
That is so cool. I will let you know the moment I get it made. Thanks!!

**Comment by DrewBrummett** [comment]
> Please feel free to provide feedback to the project.
> - What do you like?
> - What should be improved?
> - Did you build your own GameChatBalancer?
>   - Does it work?
>   - Do you need help?

How hard would it be to combine this with a macro pad? Forgive my ignorance I'm very new to the world of electronics. I've gotten into the game Guildwars 2 and that would be amazing to have. I hope to make this specific project tomorrow.
